ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -68,6 +68,12 @@ message CreatedEvent {
   // Optional
   Record create_arguments = 4;
 
+  // Opaque representation of contract payload intended for forwarding
+  // to an API server as a contract disclosed as part of a command
+  // submission.
+  // Optional
+  google.protobuf.Any create_arguments_blob = 12;
```

Comment:
It is worthwhile to ask ourselves a question how would introducing a new blob serialization schema look like. Would we be able to transparently support old client applications? Following options spring to mind:
- new schema == new field e.g. `create_arguments_blob2`
- additional field specifying the blob version -> slightly awkward and not forwards compatible
- a variation of the 2 above: new message type that combines the blob and a schema
- a version baked into the blob from the day one, this would let client not bother with it and would allow us to maintain backwards and forwards compatibility in a trivial fashion.

Reply:
Any includes a typeUrl, so in fact it includes the schema.

Reply:
Thanks for the clarification, sounds good